src/gdf_table.cuh

```diff
@@ -108,7 +82,14 @@ public:
 
   __device__ bool is_row_valid(size_type row_index) const
   {
-    return gdf::util::get_bit(d_valid_data, row_index);
```

Comment:
It looks like you undid your change to AND all of the column's valid masks together and instead are checking every columns valid bit for every call of `is_row_valid`. Was this intentional?

Reply:
So, I had run time errors in the CI environment  when I 'AND'  all of the column's bitmasks together and use it in gdf_table. So  it was on purpose to go back to a stable version where all the test pass so I can write this process considering the restriction of the CI.  I will solve this as soon as I have time. I must say that this week I have not been completely online due to meetings  and a work retreat at blazingdb.